src/v/reflection/test/async_adl_test.cc

```diff
@@ -43,7 +43,7 @@ SEASTAR_THREAD_TEST_CASE(test_async_adl_collection) {
     // Serialize
     iobuf out;
     auto original_vec = make_random_collection();
-    reflection::async_adl<ntp_vec>{}.to(out, original_vec).get();
+    reflection::async_adl<ntp_vec>{}.to(out, std::move(original_vec)).get();
```

Comment:
it doesn't seem like this would have been required by the changes. was it required?

Reply:
Nice catch, seems like i changed the interface of `adl.to( )` from `type` to `type&&`